vic/drivers/shared_image/src/vic_image_timing.c

```diff
@@ -107,7 +113,9 @@ write_vic_timing_table(timer_struct *timers,
             global_param.atmos_dt);
     fprintf(LOG_DEST, "\n");
 
-    fprintf(LOG_DEST, "  Total pes active          : %d\n", mpi_size);
+    fprintf(LOG_DEST, "  MPI Processes             : %d\n", mpi_size);
```

Comment:
1s for everything except total number of PEs available. That is still the number of available cores.